[DF-3554] - Html remove scripts from text (#92)
rapid7/insightconnect-plugins: html/help.md

|Name|Type|Default|Required|Description|Enum|
|----|----|-------|--------|-----------|----|
|doc|string|None|True|Document to transform|None|

> |remove_scripts|boolean|None|False|Remove scripts from doc as well as HTML|None|

Reviewer: ```suggestion
|remove_scripts|boolean|None|False|Remove non-HTML scripts from the document|None|
```

Reviewer: Just need to update this
